documentation/src/main/doc/modules/kafka/pages/inbound.adoc

```diff
@@ -274,6 +274,31 @@ The `partitionkey` attribute is mapped to the record's key if any.
 
 include::consumer-rebalance-listener.adoc[]
 
+=== Receiving Kafka Records in Batches
+
+By default, incoming methods receive each Kafka record individually.
+Under the hood, Kafka consumer clients poll the broker constantly and receive records in batches, presented inside the `ConsumerRecords` container.
+
+In *batch* mode, your application can receive all the records returned by the consumer *poll*.
```

Comment:
"[receive all the records returned by the consumer *poll*] at the same time", "in one go" or something like that?